### PR TITLE
Joeri action logboek -- is eerste werkende versie van action logboek zonder bestaande figma styling so dont hate uwu

### PR DESCRIPTION
**Pull request by JoeriDekker**
# Definition of done checklist
this checklist is to verify if the written feature adheres to the rules

- [ ] The code is bugfree
- [ ] Feature has been manualy tested
- [ ] Tests have passed, test should be written if none-existing
- [ ] Code is neat without unnecessary code and contains clear comments
- [ ] The user interface is created according to the designs
- [ ] Is secure on base levels (hashing, etc.)

### CODE REVIEW

File: client/src/components/logboek/actionlogboek.tsx [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+                    </select>
+                </div>
+                <div className="text-center font-bold text-gray-600">
+                    {currentYear}
+                </div>
+            </div>
+            <hr className="my-4" />
+            <div className="max-h-[300px] overflow-y-auto mt-1 shadow-md">
+                {actionlog.map((log, index) => (
+                    console.log("Action log", log),
```
Comment (Failfan13) [inline client/src/components/logboek/actionlogboek.tsx]: forgor remove console.log ofn?

File: client/src/components/logboek/actionlogboek.tsx [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+                    process.env.REACT_APP_API_URL + "/api/actionlog",
+                    {
+                        headers: {
+                            Authorization: authHeader,
+                        },
+                    }
+                );
+                console.log("Action log data", res.data);
+
+                // Map the API response to the ActionLogboekState structure
```
Comment (Failfan13) [inline client/src/components/logboek/actionlogboek.tsx]: Chat wat vinden we hiervan ???

File: client/src/components/logboek/actionlogboek.tsx [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+    return (
+        <div className="justify-center">
+            <div className="grid grid-cols-4">
+                <div className="col-span-2 font-bold">
+                    Actie Logboek 
+                    <span className="text-left text-gray-400"> ({actionlog.length} opnamen)</span>
+                </div>
+                <div className="text-center">
+                    <select id="countries" className="bg-gray-50 border border-gray-300 text-gray-900 text-sm rounded-lg">
+                        <option selected>Selecteer een maand</option>
```
Comment (nietrenswens) [inline client/src/components/logboek/actionlogboek.tsx:70]: Verwerk dit middels de lang files

**Review by nietrenswens — CHANGES_REQUESTED** [review]

File: client/src/components/logboek/actionlogboek.tsx (hunk 3 above)
Comment (JoeriDekker) [reply client/src/components/logboek/actionlogboek.tsx]: fixed

**Review by nietrenswens — APPROVED** [review]
Ik mis testen, maar voor nu ok

**Pull request merged**